ui/views.py

```diff
@@ -52,6 +53,7 @@ def index(request):
     })
 
 
+@login_required(login_url="/")
```

Comment:
for some reason I had trouble if I didn't set `login_url`, I checked and confirmed that in `settings.py` it's set to `/`, but it was redirecting me to `/login/` anyway.

from this: https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.decorators.login_required I think it might be because we don't associated `settings.LOGIN_URL` with a view? not sure!